src/services/ApolloService/ApolloService.js

```diff
@@ -0,0 +1,5 @@
+import ApolloClient from 'apollo-boost';
+
+export const client = new ApolloClient({
+  uri: 'http://localhost:5000/'
```

Comment:
rename uri to base_url (or something like this) and move it to .env. or import it from .env

Reply:
moved to .env variable REACT_APP_BASE_URI